exercises/hangman/hangman_test.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+
+from hangman import Hangman
+
+
+# Tests adapted from csharp//hangman/HangmanTest.cs
+
+class HangmanTests(unittest.TestCase):
+    def test_initially_9_failures_are_allowed(self):
+        game = Hangman('foo')
+        self.assertEqual(game.get_status(), "ongoing")
```

Comment:
It is preferable to import constants (i.e. `STATUS_ONGOING`, etc) from the solution and let the student decide what those values represent. See [the test suite for `yacht`](https://github.com/exercism/python/blob/cb8a7a7619160456ed6cbe83f18d3f27d7b2985b/exercises/yacht/yacht_test.py) for a good example of this.